modules/afprog/afprog.c

```diff
@@ -271,40 +298,66 @@ afprogram_dd_format_persist_name(AFProgramDestDriver *self)
   return persist_name;
 }
 
+static gchar *
+afprogram_dd_format_persist_name(AFProgramDestDriver *self)
+{
```

Comment:
the name here should be qualified as well, you added a qualification to format_queue_persist_name() above, which
clearly associates that to the queue object itself.

but it's far from clear what this function relates to. Also, so far we only had a single persist-entry for a driver and with this change we'll have two independent entries. @lbudai is working on centralizing this logic, I am not sure having two entries would be supported by the new framework.

If that's the case, then we'll probably need to create a struct that encapsulates the entire state (queue, reload info).

Also, the afsocket module stores the entire LogWriter instance, we could probably do the same here?